keras/callbacks.py

```diff
@@ -611,6 +611,8 @@ class TensorBoard(Callback):
             for histograms computation.
         write_images: whether to write model weights to visualize as
             image in TensorBoard.
+        write_batch_performance: whether to write training metrics on batch
```

Comment:
Can we change this parameter to be `write_step` with the options `'epoch'` and `'batch'`, and improve the description?

I think `write_step` might be more clear, and doesn't break the true/false setting in the future.

Reply:
Alternately, might it be better to write both and prepend `epoch_` or `batch_` accordingly?